### ISSUE TITLE
Add APM Server support for Debian 11

### ISSUE THREAD

**Issue by chrisdistasio** [elastic/apm-server#8729]
Debian 11 was released on August 14, 2021 and is supported by stack (7.16) and Elastic Agent (8.4)

Align with Elastic Agent platform support to ensure we have supported coverage for APM Integration.

[Related Elastic Agent Ticket](https://github.com/elastic/elastic-agent/issues/235)

Test and ongoing support for APM Server on Debian 11.

**Comment by simitt** [comment]
Conduct manual smoke testing for the standalone and Fleet managed APM Server. At least the following should be tested:
* [ ] install APM Server
* [ ] Set up templates, ILM policies, ingest pipelines via APM Integration
* [ ] Enable Tail Based Sampling
* [ ] Send APM data to APM Server 
* [ ] Verify data are successfully ingested into ES

**Comment by axw** [comment]
Tested with Debian 11 AArch64 on GCP, using 8.4.0-BC1. I've tested both the Debian package (legacy), and Elastic Agent integration, using TBS with the latter. All good.

Note: I've only tested AArch64 here, and only x86_64 in https://github.com/elastic/apm-server/issues/8730 for a bit of coverage. I don't think it's necessary to test all combinations, but if anyone feels strongly about doing that, let me know.
